src/components/Planet/PlanetInfo/PlanetDescription/PlanetBio/PlanetBio.jsx

```diff
@@ -0,0 +1,9 @@
+import './PlanetBio.css';
+
+const PlanetBio = ({bio}) => {
+    return (
+        <p className="planetBio">{bio}</p>
```

Comment:
Great job! Make the class name PascalCase and merge

Reply:
done